internal/pcrs.go

```diff
@@ -88,6 +91,31 @@ func SamePCRSelection(p *pb.PCRs, sel tpm2.PCRSelection) bool {
 	return true
 }
 
+// SamePCRSelectionDirect checks if the Pcrs has the same PCRSelection as the
+// provided given tpml.PCRSelection (including the hash algorithm).
```

Comment:
nit: no need for given, just "// provided tpml.PCRSelection (including the..."

Reply:
Resolved.